docs/index.rst

```diff
@@ -5,55 +5,59 @@
-<https://ansible-runner.readthedocs.io/en/latest/execution_environments.html>`__
-can make use of these images.
+nodes`_. An Execution Environment contains:
 
-An Execution Environment is expected to contain:
-
-- Ansible
-- Ansible Runner
-- Ansible Collections
-- Python and/or system dependencies of:
-   - modules/plugins in collections
-   - content in ``ansible-base``
-   - custom user needs
+- Python
+- Ansible-core
+- Ansible-runner
+- Ansible Collections, if needed
+- Python packages (anything you install with ``pip``)
+- System packages (anything you install with ``dnf``, ``yum``, ``apt``, or other package managers)
+- Other content you select and install
 
 .. _control nodes:
    https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#control-node
 
-.. _ansible-runner: https://github.com/ansible/ansible-runner
-
 
 Getting Started
 ***************
 
-To get started with Ansible Builder, start by creating an
-:ref:`Execution Environment definition<Definition:Execution Environment Definition>`. This
-file allows you to specify which content you would like to include in your
-execution environment, such as collections, Python requirements, and
-system-level packages.
+To get started with Ansible Builder, you must install the ``ansible-builder`` utility and a containerization tool. Once you have the tools you need, create a file called ``execution_environment.yml``. This file is a :ref:`Execution Environment definition<Definition:Execution Environment Definition>`, where
+you can specify the exact content you want to include in your
+execution environment. You can specify these items:
+- system packages (any packages installed with ``apt`` or ``yum``)
+- the version of Python
+- all Python packages (any packages installed with ``pip``)
+- the version of ansible-core
+- all Ansible Collections (anything installed with ``ansible-galaxy``)
+- other items to download, install, or configure
 
-Once you have created a definition, read through the :ref:`Usage:CLI Usage`.
+Ansible Builder reads the ``execution_environment.yml`` file and can execute two separate steps. The first step is to create a containerfile and a build context. The second step is to run a containerization tool (Podman or Docker) to build an image. You can use ``ansible-builder build`` to run both steps. Or you can use ``ansible-builder create`` to run only the first step. For more details, read through the :ref:`Usage:CLI Usage`.
 
 Collection Developers
 ^^^^^^^^^^^^^^^^^^^^^
 
-Collection developers can declare requirements for their content by providing
-the appropriate metadata. For more information, see
-:ref:`collection_metadata:Collection-level Metadata`.
+When Ansible Builder installs Collections into an Execution Environment, it also installs each Collection's dependencies. You can learn to correctly declare dependencies for your collection from the :ref:`collection_metadata:Collection-level Metadata` page.
```

Comment:
```suggestion
When Ansible Builder installs Collections into an execution environment, it also installs each Collection's dependencies. You can learn to correctly declare dependencies for your collection from the :ref:`collection_metadata:Collection-level Metadata` page.
```

Reply:
what do you think about changing 'You can learn...' to "Collection owners can learn"  .. to separate the 'you' of a person building an EE from a person who created the collection(s)?

Reply:
Good idea.